src/Symfony/Component/Cache/Adapter/PhpFilesAdapter.php

```diff
@@ -31,8 +31,8 @@ public function __construct(string $namespace = '', int $defaultLifetime = 0, st
         self::$startTime = self::$startTime ?? $_SERVER['REQUEST_TIME'] ?? time();
         parent::__construct('', $defaultLifetime);
         $this->init($namespace, $directory);
-
-        $e = new \Exception();
-        $this->includeHandler = function () use ($e) { throw $e; };
+        $this->includeHandler = function ($type, $msg, $file, $line) {
```

Comment:
shouldn't this be a static function?

Reply:
it would be a public one - better not (and would be slower if that matters)

Reply:
I meant a static anonymous function, i.e just adding `static` in front of the current `function` keyword, not moving it to a static method. It doesn't affect the visibility does it?

Reply:
static it is now